_posts/2020-10-02-status-update.md

```diff
@@ -0,0 +1,88 @@
+- [`puppetlabs-kubernetes#443`][puppetlabs-kubernetes-pr-443]: We can now specifically set the advertise address for `etcd` now thanks to [faxm0dem][faxm0dem] 
+- [`puppetlabs-postgresql#1195`][puppetlabs-postgresql-pr-1195]: Thank you [osijan][osijan] for adding `hostgssenc` as an additional option in the `$type` parameter within `postgresql::server::pg_hba_rule`
+- [`puppetlabs-stdlib#1130`][puppetlabs-stdlib-pr-1130]: A very useful additional function for parsing HOCON files from [reidmv][reidmv] - thanks for the new function!
+- [`puppetlabs-stdlib#1129`][puppetlabs-stdlib-pr-1129]: Thank you [b4ldr][b4ldr] for the new types added to `Stdlib::Ensure::File`
+- [`puppetlabs-stdlib#1128`][puppetlabs-stdlib-pr-1128]: ...and again from [b4ldr][b4ldr] - this is a great addition: we now have types aligned with the IANA port category definitions, for easier comprehension.
+- [`puppetlabs-apache#2071`][puppetlabs-apache-pr-2071]: Thanks to [creativefre][creativefre] for the fix in `apache::mod::fcgid` to make it compatible on RHEL 8.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-stdlib`][puppetlabs-stdlib] (`6.5.0`)
+- [`puppetlabs-docker`][puppetlabs-docker] (`3.12.0`)
+- [`puppetlabs-postgresql`][puppetlabs-postgresql] (`6.8.0`)
+- [`puppetlabs-mysql`][puppetlabs-mysql] (`10.7.1`)
+- [`puppetlabs-package`][puppetlabs-package] (`1.3.0`)
+
+## CI Improvements
+
+As mentioned in the last week's status update, we are currently under development of a solution that would improve the CI experience both for us as the maintainers of the modules and for everyone else in the community. One of the goals of this activity is to allow you, as the contributors, to see if any code changes affect module functionality across the supported platforms.
+At the moment we depend on Travis and Docker containers to offer you a part of this experience. The solution we are working on will harness the power of different cloud providers to allow acceptance test runs on more platforms with Puppet Litmus.
+
+## Life of an intern at Puppet
+
+As you all know already, we have our lovely new intern, [Disha][disha-maker] who is sharing her experience with the team on the [Life of an intern at Puppet]({% link _docs/life_of_intern.md %}) page. Please follow her blog to keep up to date with her activity here.
+
+## #everyonecancontribute podcast
+
+[DavidS](https://github.com/DavidS) participated in the [#everyonecancontribute](https://everyonecancontribute.com/) podcast ([recording](https://www.youtube.com/watch?v=6c0Vfn8BIM4) on Wednesday where we (amongst other things) tried out the [gitlab](https://about.gitlab.com/) integration of [gitpod.io](https://www.gitpod.io/), which provides a fully online hosted VSCode instance to code in any browser. A lot of things just work out of the box, and we even got some of the [Puppet VSCode extension](https://puppet-vscode.github.io/) to work.
```

Comment:
@DavidS You can link to https://everyonecancontribute.com/post/2020-09-30-kaeffchen-20-gitpod/ as well which has lots of details in written English.

Reply:
Thanks! Added an edit